spec/factories/external_post.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :external_post do
+    title 'Example Discussion'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.